models/Asset/Image/Thumbnail/Config.php

```diff
@@ -848,14 +848,15 @@ public function generateAutoName(): void
     /**
      * @internal
      */
-    public function getHash(): string
+    public function getHash(array $params = []): string
     {
         return hash('xxh32', serialize([
             $this->getPreserveAnimation(),
             $this->getQuality(),
             $this->isPreserveColor(),
             $this->isPreserveMetaData(),
             $this->getItems(),
+            $params,
```

Comment:
Should we unpack it?

```suggestion
            ...$params,
```

Reply:
any difference since we serialize anyway? 🤔 But I would be fine with it, if there's a benefit 😊

Reply:
Probably not.